### DISCUSSION TITLE
Screen blank and no info from board ESP32-3248S035 capacitive

### DISCUSSION THREAD

**Discussion by dcknuth** [rzeldent/esp32-smartdisplay#195] [Q&A]
I was able to get the demo cloned, compiled and uploaded to the board. However the board is completely unresponsive and blank after that. I targeted just my board by uncommenting this line:
default_envs = esp32-3248S035C
in the platformio.ini
I also tried setting:
#define LV_USE_ST7796		1
in lv_conf.h and deleting and rebuilding the .pio directory, but no change. Any suggestions of things to try? It seems I should get some feedback from the board about stats, but I didn't see any, but not sure if that would just appear in the console after loading via USB.

**Comment by rzeldent** [comment]
Duplicate: https://github.com/rzeldent/platformio-espressif32-sunton/issues/61
